sdf/final-project/presentation.md

```diff
@@ -59,6 +69,8 @@ This is intended to be filled out by staff members of DPI Tech Apprenticeship pr
 - [ ] **Eye Contact**: Does the speaker effectively use eye contact?
 - [ ] **Owning the Space**: Does the speaker confidently utilize the space?
 - [ ] **Addressing Questions**: Is the presenter able to effectively respond to questions, showcasing a deep understanding of their project and its context?
+
+<!-- Maybe move to it's own section. Looking for a slide or section that discusses a key technical decision, aspect, or hurdle of their project. (eg cloud storage, domain design, external api, etc.) -->
 - [ ] **Technical Fluency**: Does the presenter display fluency in discussing the technical aspects of their project, able to articulate the significance of their choices and challenges without resorting to jargon?
```

Comment:
Should we also notate if this has to be stated during the presentation and not during Q&A time. Or are we going to give them credit if they talk about a technicality during questions.

Reply:
Matthew made an interesting point about asking the same questions. I do think if we're giving credit for stuff they share in response to a question then we need to be sure to provide the same opportunity to all with the same questions.